File: Day2/Solution.java
public class Arithmetic {

    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        double mealCost = scan.nextDouble(); // original meal price
        int tipPercent = scan.nextInt(); // tip percentage
        int taxPercent = scan.nextInt(); // tax percentage
        scan.close();

        // Write your calculation code here.
        mealCost=mealCost+(mealCost*tipPercent/100)+(mealCost*taxPercent/100);
        // cast the result of the rounding operation to an int and save it as totalCost
        int totalCost = (int) Math.round(mealCost);

        // Print your result
        System.out.print("The total meal cost is "+totalCost+" dollars.");
    }
}

File: Day2/Solution.swift
import Foundation

// Complete the solve function below.
func solve(meal_cost: Double, tip_percent: Int, tax_percent: Int) -> Void {
    let tip = (meal_cost * (Double(tip_percent)/100))
    let tax = (meal_cost * (Double(tax_percent)/100))
    let mealCost = meal_cost + tip + tax

    // cast the result of the rounding operation to an int and save it as totalCost
    let totalCost = Int(round(mealCost))

    // Print your result
    print(totalCost)
}
